ci/helm/preview/values.yaml

```diff
@@ -0,0 +1,93 @@
+expose:
+  Annotations:
+    helm.sh/hook: post-install,post-upgrade
+    helm.sh/hook-delete-policy: hook-succeeded
+  config:
+    # add tls configuration
+    domain: platform.dev.nuxeo.com
+    exposer: Ingress
+    http: true
+    tlsacme: true
+    tlsSecretName: platform-cluster-tls
+    urltemplate: '"{{.Service}}-{{.Namespace}}.{{.Domain}}"'
+
+cleanup:
+  Args:
+    - --cleanup
+  Annotations:
+    helm.sh/hook: pre-delete
+    helm.sh/hook-delete-policy: hook-succeeded
+
+nuxeo:
+  fullnameOverride: preview
```

Comment:
What about `nuxeo-preview` to have it very clear in the pod list that it's a nuxeo preview (even if the pod are in a dedicated namespace)?

Reply:
I propose preview-nuxeo to stay consistent with the names of all the other pods. 
And It would ease autocomplete if you switch from a pod to another (no need to erase "nuxeo-preview" but just "nuxeo" then type "elas..." for example